packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -96,43 +117,45 @@ export const getComponentCss = (
     //     display: block,
     //   }
     header: {
-      display: 'grid',
-      gap: gridGap,
```

Comment:
Hmm, why remove? Now the text can overlap the prev/next buttons.
<img width="452" alt="Bildschirm­foto 2023-02-06 um 10 55 47" src="https://user-images.githubusercontent.com/4380295/216941328-cda5dce5-633f-43c8-a7e4-e1f6d7acd810.png">

Reply:
Fixed it